Idno/Common/Entity.php

```diff
@@ -1427,6 +1557,23 @@ function getShortDescription($words = 25)
             return $description;
         }
 
+        /**
+         * Retrieve a short description of this page suitable for including in page metatags
+         * @param $words Number of words to limit to, if we're generating a short description on the fly (default: 25)
+         * @return string
+         */
+
+        function getFormattedContent()
```

Comment:
So function's purpose is the intention behind the existing `getShortDescription()` method - I wonder if we need to reconcile that function to this?

Reply:
With its name `getShortDescription()` and the fact that it both strips tags, as well as gets the `Article` subtitle field, I think it might be best to keep it separate, at least for now !